docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -8,7 +8,7 @@
     "# Direct DAAC S3 Bucket Access (BETA)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    with s3_fsspec_rp.open(nsidc_object, "rb") as f:
</pre><p>is this supposed to be <code>s3_fsspec_requesterpays</code></p><p> </p><p>Also wondering if we can find an example that loads faster (that might be the dmr++ question for later) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>